pkg/policymutation/policymutation.go

```diff
@@ -38,16 +38,19 @@ func GenerateJSONPatchesForDefaults(policy *kyverno.ClusterPolicy, autogenIntern
 		updateMsgs = append(updateMsgs, updateMsg)
 	}
 
-	patch, errs := GeneratePodControllerRule(*policy, autogenInternals, log)
-	if len(errs) > 0 {
-		var errMsgs []string
-		for _, err := range errs {
-			errMsgs = append(errMsgs, err.Error())
-			log.Error(err, "failed to generate pod controller rule")
+	// if autogenInternals is enabled, we don't mutate rules in the webhook
+	if !autogenInternals {
```

Comment:
Can we use `toggle.AutogenInternals()` directly?

Reply:
Looks like we can. Gonna change it.

Reply:
Done.